app/src/main/java/org/oppia/android/app/testing/BottomSheetOptionsMenuTestActivityPresenter.kt

```diff
@@ -0,0 +1,53 @@
+package org.oppia.android.app.testing
+
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.test_bottom_sheet_options_menu.*
+import org.oppia.android.R
+import org.oppia.android.app.help.HelpActivity
+import org.oppia.android.app.model.ReadingTextSize
+import org.oppia.android.app.options.OptionsActivity
+import org.oppia.android.app.player.exploration.BottomSheetOptionsMenu
+import org.oppia.android.app.utility.FontScaleConfigurationUtil
+import javax.inject.Inject
+
+private const val INTERNAL_PROFILE_ID = 0
+
+/** The presenter for [BottomSheetOptionsMenuTestActivity] */
+class BottomSheetOptionsMenuTestActivityPresenter @Inject constructor(
+  private val activity: AppCompatActivity,
+  private val fontScaleConfigurationUtil: FontScaleConfigurationUtil
+) {
+
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.test_bottom_sheet_options_menu)
+    activity.action_bottom_sheet_options_menu.setOnClickListener {
+      val bottomSheetOptionsMenu = BottomSheetOptionsMenu()
+      bottomSheetOptionsMenu.showNow(activity.supportFragmentManager, bottomSheetOptionsMenu.tag)
+    }
+  }
+
+  fun handleOnOptionsItemSelected(itemId: Int): Boolean {
```

Comment:
Instead of intenting here, could we maybe introduce an interface & a mock, then call it to pass the item ID to the test? References:

https://github.com/oppia/oppia-android/blob/d22f502dbf01b21dd9b54002dd564b23f1c12043/app/src/main/java/org/oppia/android/app/notice/testing/BetaNoticeDialogFragmentTestActivity.kt#L24

https://github.com/oppia/oppia-android/blob/1fcd7746f23af5e546558203eb685d0c7662a39d/app/src/sharedTest/java/org/oppia/android/app/notice/BetaNoticeDialogFragmentTest.kt#L205

https://github.com/oppia/oppia-android/blob/1fcd7746f23af5e546558203eb685d0c7662a39d/app/src/sharedTest/java/org/oppia/android/app/notice/BetaNoticeDialogFragmentTest.kt#L173

Reply:
Removed this test and added mock in new `BottomSheetMenuTestActivity`